packages/camera/camera_avfoundation/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -733,6 +741,25 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
```

Comment:
I'm pretty sure checking these it will break platform tests on stable (which we don't run in presubmit). We've been leaving the projects not referencing SPM until after the next stable.

Reply:
Sorry, this is just OCMock. Ignore me.